docs/magento-layout-loader.md

```diff
@@ -0,0 +1,104 @@
+# magento-layout-loader
+
+_This is a very early implementation, and the API should be considered
+unstable._
+
+The `magento-layout-loader` is a [webpack
+loader](https://webpack.js.org/loaders/) that implements the Magento PWA Studio
```

Comment:
What about:
> ... uses React to translate the Magento PWA Studio layout language into HTML content"

Reply:
Still ends up being a bit misleading. Technically, this plugin doesn't use React at all.

I agree this could be worded better, just struggling to find the right words.